tests/symengine/sd_common_tests/batch_optimizer_02.h

```diff
@@ -123,11 +103,29 @@ test_scalar(const int n_runs, TimerOutput &timer)
         const NumberType val_s     = optimizer.evaluate(symb_s);
         const NumberType val_ds_dx = optimizer.evaluate(symb_ds_dx);
         if (i == 0)
-          deallog << "evaluation: "
-                  << "  s: " << val_s << "  ds_dx: " << val_ds_dx << std::endl;
+          std::cout << "evaluation: "
+                    << "  s: " << val_s << "  ds_dx: " << val_ds_dx
+                    << std::endl;
+
+        // The result is not stable and depends on standard library, number
+        // type, optimization parameters and compiler being used. However, we
+        // also don't want to maintain a whole bunch of blesses output
+        // variants. So we set a loose tolerance here.
+        static const double tol = 1e-3;
```

Comment:
@tamiko I used some pretty permissive tolerances, but admittedly I have not normalised the values because I was lazy. Can you please test this branch to see if it now works for you? If you still find this to be an issue then I'll normalise the values and we can try again.